ReactNativeClient/lib/components/BackButtonDialogBox.ts

```diff
@@ -0,0 +1,24 @@
+const { BackButtonService } = require('lib/services/back-button.js');
+const DialogBox = require('react-native-dialogbox').default;
+
+export default class BackButtonDialogBox extends DialogBox {
```

Comment:
Created this so that it can be reused in a few other places as mentioned in https://github.com/laurent22/joplin/issues/3364